src/cpp/runner/runnerapp.cpp

```diff
@@ -654,7 +651,7 @@ class RunnerApp::Private : public QObject
 				s->error.connect(boost::bind(&Private::service_error, this, boost::placeholders::_1, s))
 			};
 
-			if(!args.mergeOutput || qobject_cast<Mongrel2Service*>(s))
+			if(!args.mergeOutput)
```

Comment:
hmm, guess we broke the casting in an earlier commit. I suggest adding a `bool alwaysLogStatus() const { return false; }` method to `Service` that `Mongrel2Service` overrides to return true, and then call that method here instead of casting.